arcgis-ios-sdk-samples/Layers/Dictionary renderer with feature layer/DictionaryRendererViewController.swift

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DictionaryRendererViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            // Initialize the map.
+            mapView.map = AGSMap(basemap: .topographic())
+            
+            // Display symbology using a geodatabase.
+            displayLayersFromGeodatabase()
+        }
+    }
+    
+    func displayLayersFromGeodatabase() {
+        // Load geodatabase from shared resources.
+        let geodatabaseURL = Bundle.main.url(forResource: "militaryoverlay", withExtension: ".geodatabase")!
+        let generatedGeodatabase = AGSGeodatabase(fileURL: geodatabaseURL)
+        
+        // Instantiate and load symbol dictionary from shared resources.
+        let styleURL = Bundle.main.url(forResource: "mil2525d", withExtension: "stylx")
+        let dictionarySymbol = AGSDictionarySymbolStyle(url: styleURL!)
+        dictionarySymbol.load { [weak self] (error: Error?) in
+            if let error = error {
+                self?.presentAlert(error: error)
+            } else {
+                return
+            }
+        }
+        
+        // Once geodatabase is done loading, create feature layers from each feature table.
+        generatedGeodatabase.load { [weak self] (error: Error?) in
+            if let error = error {
+                self?.presentAlert(error: error)
+            } else {
+                for featureTable in generatedGeodatabase.geodatabaseFeatureTables {
+                    // Create a feature layer from the feature table.
+                    let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+                    // Set the minimum visibility scale.
+                    featureLayer.minScale = 1000000
+                    featureLayer.load { [weak self] (error: Error?) in
```

Comment:
FYI: A closure will capture the same kind of reference to `self` as the enclosing scope by default. Since `self` is `weak` in the enclosing scope, this closure will weakly capture `self` without needing to specify that in the capture list.

```suggestion
                    featureLayer.load { (error: Error?) in
```